modules/transport-netty4/src/main/java/org/elasticsearch/http/netty4/Netty4HttpServerTransport.java

```diff
@@ -313,6 +315,7 @@ protected void initChannel(Channel ch) throws Exception {
                 ch.pipeline().addLast("encoder_compress", new HttpContentCompressor(handlingSettings.getCompressionLevel()));
             }
             ch.pipeline().addLast("request_creator", requestCreator);
+            ch.pipeline().addLast("response_creator", responseCreator);
```

Comment:
We will probably want it at some point if the NIO transport becomes used in production. But it is not really necessary at this moment since it is mostly a test transport. I will probably follow-up at some point.